io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -151,12 +216,11 @@ def check_pass(self, cmd, errmsg):
         if process.system(cmd, ignore_status=True, shell=True) != 0:
             self.fail(errmsg)
 
-    def cc_showprogress(self):
+    def showprogress(self, cmd):
 
         """
-        Helper function to see the CC progress
+        Helper function to see progress of a particular operation
```

Comment:
It can be any of below.
CC
BGI
Rebuild
Migration
Init
PR and few other background operation. So I thought to keep it generic.